engines/director/detection_tables.h

```diff
@@ -1861,6 +1864,7 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACDEMO1_l("garyukeiba", "Demo", "Garyu Keiba Auto Demo", "276bee761e48a6fd709df77d5c2f60dd", 393175, Common::JA_JPN, 313),
 
 	MACDEMO1_l("goalrush1", "Demo", "GOAL RUSH!! DEMO", "f5277c53bacd27936158dd3867e587e2", 394984, Common::JA_JPN, 311),
```

Comment:
```suggestion
	MACDEMO1_l("goalrush1", "Demo", "GOAL RUSH!! DEMO", "f5277c53bacd27936158dd3867e587e2", 394728, Common::JA_JPN, 311),
```

Reply:
These demos are indeed different. I'll leave any other corrections to file sizes as suggested changes.